java/server/src/org/openqa/selenium/netty/server/NettyServer.java

```diff
@@ -46,19 +48,30 @@
   private final int port;
   private final URL externalUrl;
   private final HttpHandler handler;
+  private final SslContext sslCtx;
 
   private Channel channel;
 
-  public NettyServer(BaseServerOptions options, HttpHandler handler) {
+  public NettyServer(BaseServerOptions options, HttpHandler handler) 
+    throws SSLException, CertificateException {
     Objects.requireNonNull(options, "Server options must be set.");
     Objects.requireNonNull(handler, "Handler to use must be set.");
 
+    Boolean secure = options.isSecure();
```

Comment:
`boolean`: we never expect this to be `null`, and the unboxing that happens on the next line will throw an NPE if that happens.

Reply:
I think the current implementation of isSecure ensures it always is Boolean.